csrc/python_frontend/fusion_record_serde.h

```diff
@@ -0,0 +1,122 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+#include <python_frontend/factory.h>
+#include <python_frontend/fusion_record.h>
+
+namespace nvfuser::python_frontend {
```

Comment:
nitpick. maybe we should hide serialization/deserialization stuff under a namespace.

Reply:
Was the namespace added?

Reply:
Yes. They're placed under `nvfuser::serde`.